packages/code/src/renderingPlugin.tsx

```diff
@@ -385,7 +390,7 @@ export function TableContent({ lines }: TableContentProps) {
   return (
     <>
       {trimmedLines.map((line, index) => {
-        const currentLineNumber = index + 1;
+        const currentLineNumber = index + lineOffset || 1;
```

Comment:
Actually, this line is also throwing a TS error since lineOffset is possibly undefined, and you're adding it to index here. I'd recommend doing either `index + (lineOffset ?? 1)` or `index + (lineOffset || 1)` here to resolve the issue.